core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -340,8 +340,9 @@ static struct tee_mmap_region *init_xlation_table(struct tee_mmap_region *mm,
 {
 	unsigned level_size_shift = L1_XLAT_ADDRESS_SHIFT - (level - 1) *
 						XLAT_TABLE_ENTRIES_SHIFT;
-	unsigned level_size = 1 << level_size_shift;
-	uint64_t level_index_mask = XLAT_TABLE_ENTRIES_MASK << level_size_shift;
+	unsigned level_size = BIT32(level_size_shift);
```

Comment:
Checkpatch also warns here:
`WARNING: Prefer 'unsigned int' to bare use of 'unsigned'`